docs/how-tos/sensor-module.md

```diff
@@ -1,12 +1,12 @@
 ---
 title: "Create a sensor module with Python"
-linkTitle: "Create a sensor module"
+linkTitle: "Create a sensor module with Python"
 type: "docs"
 weight: 26
 images: ["/icons/components/sensor.svg"]
 icon: true
 tags: ["modular resources", "components", "services", "registry"]
-description: "Add a custom resource by creating and deploying a module to your machine."
+description: "Add support for a new model of sensor by creating a module written in Python."
```

Comment:
I wanted to emphasize the Python specificity here and I understand your point about "support" but I want to de-emphasize the "physical" bit since that's unnecessary
```suggestion
description: "Add a new model of sensor component by writing a module in Python."
```

Reply:
ok